target/linux/mediatek/patches-6.6/108-hack-adjust-Winbond-W25N01KV-spi-nand-oob.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/mtd/nand/spi/winbond.c
++++ b/drivers/mtd/nand/spi/winbond.c
+@@ -182,7 +182,7 @@ static const struct spinand_info winbond
```

Comment:
Please add a patch description. This should contain why this is needed.

Reply:
Done.